typings/artifacts.d.ts

```diff
@@ -359,8 +359,8 @@ declare global {
         processEvents: Array<TraceEvent>;
         /** The subset of trace events from the page's main thread, sorted by timestamp. */
         mainThreadEvents: Array<TraceEvent>;
-        /** The event marking the start of tracing in the target browser. */
-        startedInPageEvt: TraceEvent;
+        /** Various ids for the current trace context. */
```

Comment:
maybe something like `/** IDs for the trace's main frame, process, and thread. */`?